pkg/services/alerting/notifiers/pagerduty.go

```diff
@@ -99,6 +95,26 @@ func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
 	pn.log.Info("Notifying Pagerduty", "event_type", eventType)
 
 	payloadJSON := simplejson.New()
+	bodyJSON := simplejson.New()
```

Comment:
I don't see the reason for moving the `bodyJSON` declaration here, since it's not used until later on in any case.

Reply:
not sure why I did that, so I'll move back